db/change_cache_test.go

```diff
@@ -1001,6 +1001,9 @@ func TestLowSequenceHandlingWithAccessGrant(t *testing.T) {
 //     - since it's been terminated, should return error before executing a second view query
 func TestChannelQueryCancellation(t *testing.T) {
 
+	if !base.UnitTestUrlIsWalrus() {
+		t.Skip("Skip test with LeakyBucket dependency test when running in integration")
```

Comment:
I'm curious what doesn't work with `LeakyBucket` on Couchbase Server but will merge just to get more green builds in the meantime.